src/plugins/controls/public/control_group/control_group.scss

```
@@ -54,6 +57,21 @@ $controlMinWidth: $euiSize * 14;
       cursor: grabbing;
     }
   }
+
+  @media only screen and (max-width: 767px) {
```

```suggestion
  @include euiBreakpoint('xs', 's', 'm') {
```

I believe this is the EUI way to achieve this